pkg/networkpolicy/controller_test.go

```diff
@@ -125,3 +127,70 @@ func newTestController() *networkpolicyController {
 		informersFactory.Core().V1().Nodes().Informer().GetStore(),
 	}
 }
+
+func TestConfig_Defaults(t *testing.T) {
+	tests := []struct {
+		name     string
+		config   Config
+		expected Config
+	}{
+		{
+			name: "empty",
+			config: Config{
+				NodeName: "testnode", // nodename defaults to os.Hostname so we ignore for tests
```

Comment:
hmm the comment is confusing probably because I miss context, if "testnode" is an explicit override you are providing, then we are not ignoring for tests right?

Reply:
if is set, then it will not be defaulted :) .. the test is about testing the Defaults are applied

Reply:
ack so testing os.Hostname is ignored is what you mean given we'd need mocks etc so you do the overrides.. gotcha!

Reply:
yeah, not great comment and good review, but I think we can live with it

Thanks